aviatrix/utils.go

```diff
@@ -249,18 +242,18 @@ func versionFormat1(version1, version2, flag string) bool {
 	return compareVersion(reg.ReplaceAllString(versionArray1[0], ""), reg.ReplaceAllString(versionArray2[0], ""))
 }
 
-// this function can sort the firewall_image_version format like: PA-VM-10.1.0
+// versionFormat2 sorts the firewall_image_version format like: PA-VM-10.1.0
```

Comment:
Should be `versionFormat2 compares...`

Reply:
This one could also be more descriptive. Something like `comparePAVersion`

Reply:
thank you, have fixed: comparePAVersion compares firewall_image_version format like: PA-VM-10.1.0